pgx/src/spi.rs

```diff
@@ -487,6 +488,48 @@ impl SpiTupleTable {
             }
         }
     }
+
+    /// Returns the number of columns
+    pub fn columns(&self) -> Option<i32> {
```

Comment:
Couldn't this return `usize` unconditionally? If there's no `tupdesc` then there are 0 columns.

Reply:
I guess this is reasonable. I've pushed an update.